.github/2.0/cookbooks/Document.md

```diff
@@ -467,7 +469,7 @@ You can add **chunks** (sub-Document) and **matches** (neighbour-Document) to a
   d.matches.append(Document())
   ```
 
-Note that both `doc.chunks` and `doc.matches` return `DocumentArray`, which we will introduce later.
+Note that both `doc.chunks` and `doc.matches` return `ChunkArray` and `MatchArray`, which are sub-classes of `DocumentArray`. We will introduce `DocumentArray later.
```

Comment:
We could add a link to the respective section here. Is there an opinion about linking in the Cookbooks already?

Reply:
done